src/compiler/scala/tools/nsc/ast/parser/Scanners.scala

```diff
@@ -1094,10 +1094,10 @@ trait Scanners extends ScannersCommon {
         syntaxError("Invalid literal number")
     }
 
-    @inline private def isNumberSeparator(c: Char): Boolean = c == '_' //|| c == '\''
+    @inline private def isNumberSeparator(c: Char): Boolean = c == '_'
 
     @inline private def removeNumberSeparators(s: String): String =
-      if (s.indexOf('_') > 0) s.replaceAllLiterally("_", "") /*.replaceAll("'","")*/ else s
```

Comment:
We could have had `1'000'000`? Now that `Symbol` is gone what's there left for poor `'`? Other than character literals, of course. I guess it's not so much gone as just quietly not reinvited.

Reply:
Ichoran wanted `1'000`. There's also `x'` for redefining `x`.